style-dictionary-utils/transforms/value_shadow.js

```diff
@@ -11,8 +11,8 @@ module.exports = {
     const { x, y, blur, spread, color, opacity } = prop.value;
 
     // convert hex code to rgba string
-    const shadowColor = tinycolor(color.value);
-    shadowColor.setAlpha(opacity.value);
+    const shadowColor = tinycolor(color.value ?? color);
+    shadowColor.setAlpha(opacity.value ?? opacity);
```

Comment:
This fixed the issue with how the shadow styles were generated.

Reply:
good catch and thanks for fixing!